app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -6,11 +6,13 @@
     init: function init(params) {
       this.$form = $(params.selector);
       this.rightToLeftLocales = params.right_to_left_locales;
+      this.newsTypeSelect = $('select#edition_news_article_type_id')
```

Comment:
Looks like the existing JS prior to this was using `$varName` to denote jQuery dom elements. If we're going to have that then we should use it consistently in the new stuff that we have added.